src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/DatabaseTreeNode.cs

```diff
@@ -28,14 +28,15 @@ public DatabaseTreeNode(ServerNode serverNode, string databaseName): this()
             // We detect that here and fall back to master
             if (db.State == SqlSmoState.Creating && !IsDWGen3(db))
             {
```

Comment:
log a bug to only do this fallback for non-Azure connections

Reply:
https://github.com/microsoft/sqltoolsservice/issues/1767